### DISCUSSION TITLE
Double-click repeater and block headers to collapse/uncollapse

### DISCUSSION THREAD

**Discussion by kirkbushell** [filamentphp/filament#7665] [Ideas]
I've been migrating away from Twill CMS, and honestly been blown away by how good Filament is and how easily it covers my various use-cases, often making some requirements much easier to implement.

One thing I miss from Twill CMS is the double-click on the headers to collapse/uncollapse that block or repeater. Seems a small thing, but it's much easier to go up to the header and double-click to toggle the collapsed state than to go to the right hand side (right next to the delete, mind you) to collapse it.

I think this would be pretty easy to support, too - and would happy to create a PR to solve it.

**Comment by danharrin** [comment]
You can do a PR if you want, but honestly I'm not sure how intuitive a double click is

**Reply by kirkbushell** [reply]
This is how Twill CMS does it, they add a image to that header that looks like a bunch of dots (2 rows) that can be used to move the repeater, or, it can be double-clicked.

It's actually very intuitive. I wasn't taught that's what it did and figured it out.

**Comment by sinnbeck** [comment]
Another idea would be to just make your own class that extends the repeater and alter the header on that. I have extended my own Rich editor to add more features. Or make a plugin

**Reply by kirkbushell** [reply]
Yeah, for sure - I could. How did you alter the header specifically though?
